_includes/why_d4a.md

```diff
@@ -0,0 +1,124 @@
+The [Docker for
+{{cloudprovider}}](https://docs.docker.com/docker-for-{{cloudprovider |
+downcase}}/) project was created and is being actively developed to ensure that
+Docker users can enjoy a fantastic out-of-the-box experience on
+{{cloudprovider}}.  It is now generally available and can now be used by
+everyone.
+
+As an informed user, you might be curious to know what this project has to offer
+you for running your development, staging, or production workloads.  Here are
+some talking points for you to consider.
+
+## Native to Docker
+
+While some container orchestration offerings seem to require a PhD to operate
+successfully, or perhaps layer unneeded cruft on top of the Docker CLI, Docker
+for {{cloudprovider}} was built with a passion for simplicity and great user
+experience. We believe strongly that additional complexity must be justified by
+substantial additional benefit.
+
+Instead of having to learn one or more layers on top of Docker, with Docker for
+{{cloudprovider}} you will interact with Docker (including native Docker
+orchestration) directly.  These setups free you to focus on the thing that
+matters most: running your workloads. This will help you and your team to
+deliver more value to the business faster, to speak one common "language", and
+to have fewer details to keep in your head at once.
+
+The skills that you and your team have already learned, and will continue to
+learn, using Docker on the desktop or elsewhere will automatically carry over to
+using Docker on {{cloudprovider}} and vice versa.  The added consistency across
+clouds also helps to ensure that a migration or multi-cloud strategy would be
+easier to accomplish in the future if desired.
+
+## Skip the Boilerplate and Maintenance Work
+
+While some might find joy in spending weeks or months meticulously writing and
```

Comment:
Yes, it's a little snarky :-) - it's great for a blog post, but maybe not optimal for documentation